### PR TITLE
Add WatchOS 2 support.

### PR DESCRIPTION
**Pull request by e2l3n**
<b>Description</b>: 
Update `MMWormhole` API to latest version [2.0.0](https://github.com/mutualmobile/MMWormhole/releases). 
WatchOS 2 SDK has introduced a new API that facilitates communication between a WatchKit extension and its companion iOS app. 
See this https://github.com/mutualmobile/MMWormhole/issues/54 issue for more details on the topic.
<b>Details</b>: 
Rebuilt the `libmmwormhole.a` static lib with latest `MMWormhole` API (version [2.0.0](https://github.com/mutualmobile/MMWormhole/releases)). Add implementation that supports watch apps built with WatchOS 2 SDK and iOS 9.
<b>TeamPulse</b>: 
[[iOS][AppBuilder][Cordova] Add WatchOS 2 support to AppleWatch plugin.](http://teampulse.telerik.com/view#item/305755)

### CODE REVIEW

**Comment by e2l3n** [comment]
ping @EddyVerbruggen @galexandrov @ligaz

**Comment by e2l3n** [comment]
Squash commits.

**Comment by galexandrov** [comment]
:+1:  if it works in Cordova and AB cases

File: src/ios/app/AppleWatch.m [hunk 1]
```diff
@@ -17,9 +21,21 @@ @implementation AppleWatch
...
     NSString *appGroup = [NSString stringWithFormat:@"group.%@", [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleIdentifier"]];
-
+    
+#if ( defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 90000 )
+    self.watchConnectivityListeningWormhole = MMWormholeSession.sharedListeningSession;
+    // Make sure we are activating the listening wormhole so that it will receive new messages from
+    // the WatchConnectivity framework.
+    [self.watchConnectivityListeningWormhole activateSessionListening];
+    // Initialize the wormhole using the WatchConnectivity framework's application context transiting type
+    self.wormhole = [[MMWormhole alloc] initWithApplicationGroupIdentifier:@"group.com.mutualmobile.wormhole"
```
Comment (EddyVerbruggen) [inline src/ios/app/AppleWatch.m]: I _think_ this should not be a fixed groupIdentifier but rather the value of `appGroup` a few some 10 lines back.

**Comment by EddyVerbruggen** [comment]
Hey @e2l3n I had 1 review comment, can you take a look when you have time please?

@galexandrov are you able to test the hooks with this new release? I can do the CLI hooks, but your cool AB hooks are a different story..

**Comment by e2l3n** [comment]
@EddyVerbruggen thanks, this has been fixed now. Regarding the AB hooks - I'm testing it and will give an update soon.

**Comment by NorthMcCormick** [comment]
@e2l3n Any updates on this? I'm looking to use your branch but if there is a chance of this being merged soon I might just wait.

**Comment by galexandrov** [comment]
Unfortunately the PR is not quite correct and will not work as expected, so we will close this one and soon we will open a new PR for WatchOS 2 support

**Pull request closed without merging**